### ISSUE TITLE
Support Android API < 28

### ISSUE THREAD

**Issue by adamglin0** [adamglin0/compose-shadow#1]
Now in Android API 28

<img width="321" alt="image" src="https://github.com/user-attachments/assets/bad14937-3450-4054-848a-a1acd1c21085">

**Comment by adamglin0** [comment]
https://developer.android.com/reference/android/graphics/BlurMaskFilter

**Comment by adamglin0** [comment]
android version < 28 not support BlurMaskFilter, so I will check the implement of Modifier.shadow firstly

**Comment by adamglin0** [comment]
[imla](https://github.com/desugar-64/imla)

GPU-Accelerated Blurring for Android Jetpack Compose UI

[haze](https://github.com/chrisbanes/haze)
Compose Multiplatform / Jetpack Compose Blur Modifier

**BUT NOW, HAZE NOT SUPPORT BLUR BELOW ANDROID28**

**Comment by Skaldebane** [comment]
@adamglin0 You might wanna look at https://github.com/GIGAMOLE/ComposeShadowsPlus/.

And blur (like in Haze) is only available on SDK 32 and above.

**Comment by adamglin0** [comment]
> Instead of failing with a build error:
> 
> ```
> Error: uses-sdk:minSdkVersion 24 cannot be smaller than version 28 declared in library [com.adamglin:compose-shadow-android:2.0.1]
> ```
> 
> I would gladly accept no shadows on old Androids -- who cares?

You are right, the problem is that I set minSDK for `compose-library` ; I will remove this unnecessary restriction.

**Comment by adamglin0** [comment]
COMPOSE WILL SUPPORT DROPSHADOW AND INNERSHADOW.
https://android-review.googlesource.com/c/platform/frameworks/support/+/2594888

**Comment by Skaldebane** [comment]
Super cool!

That'll probably be the fastest deprecation of a Compose library 😆
But (hopefully) it was part of the reason that pushed them to natively add it!
